index.js

```diff
@@ -1,5 +1,5 @@
 import app from "./src/app.js";
 
 app.listen(5080, () => {
-    console.log('listening on port 5080')
+  console.log("listening on port 5080");
```

Comment:
This change seems to have happened due to automatic formatting, and that not everyone in the team have the same settings. This leads to changes in the code that don't "mean" anything, which can be confusing for reviewers. I recommend two things in future projects:

1. Make sure all team members have the same settings, e.g. using a tool like [Prettier](https://prettier.io)
2. If differences occur, coordinate within the team to make sure you have the same settings (1) and then create a separate PR that fixes all files in one go